cmd/kubeturbo/app/kubeturbo_builder.go

```diff
@@ -57,7 +57,7 @@ const (
 	defaultDiscoveryIntervalSec       = 600
 	DefaultValidationWorkers          = 10
 	DefaultValidationTimeout          = 60
-	DefaultDiscoveryWorkers           = 4
+	DefaultDiscoveryWorkers           = 10
```

Comment:
How will we know that these changes work if the two culprit nodes are back up in BMO?

Reply:
For the discovery worker change, we can tell from the log immediately: 
I0729 17:54:07.519167    6430 k8s_discovery_client.go:63] Number of discovery workers: 10.

For the concurrency handling, the fact is we are removing the unnecessary go routine (which caused the deadlock), so for sure we won't see the deadlock anymore.